package.json

```diff
@@ -34,22 +34,22 @@
     "@storybook/addon-storysource": "^3.4.12",
     "@storybook/react": "~3.4.12",
     "@storybook/storybook-deployer": "^2.0.0",
-    "@theforeman/vendor-dev": "^1.7.0",
+    "@theforeman/vendor-dev": "^3.0.0",
     "argv-parse": "^1.0.1",
     "axios-mock-adapter": "^1.10.0",
-    "babel-cli": "^6.10.1",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^7.1.1",
+    "@babel/cli": "^7.7.0",
```

Comment:
do we also need to package those? because we don't have babel 7.x packaged so far

Reply:
or is this just for dev setup?

Reply:
Didn't you packaged any `babel` before? You need them for the webpack build

Reply:
Notice there is a WIP to move them to a separate package:
https://github.com/theforeman/foreman/pull/7178

Reply:
The babel packages changed their names from v6 to v7
It used to be `babel-core` and now it's `@babel/core`

Reply:
and we only have babel-core (and friends) in 6.26 packaged, not 7.x

Reply:
True